code/src/SureQuestion.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+export const SureQuestion = ({ sure, setSure}) => {
+    const areYouSure = ["Yep! Pretty sure.", "Dunno", "Maybe not"];
+
+    return (
+      <div>
+        <h2>Is it really your favourite?</h2>
+        {areYouSure.map(sure => (
+          <label tabIndex="0">
+            <input
+              type="radio"
+              value={sure}
```

Comment:
For radio buttons we need to use `checked` attribute instead of `value`.
Right now I am able to check all 3 radio buttons - it doesn't make sense, only one of them should be selected at the time